gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,54 @@ def get_possible_database_path():
     return path
 
 
+def create_possible_database_path():
+    """Create the directory to what is possibly a GRASS Database.
+
+    Create directory named grassdata in the usual locations.   
+    
+    Returns the new path as a string.  
+    """
+    home = os.path.expanduser('~') 
+    tmp = os.path.join(tempfile.gettempdir(),
+                               "grassdata_{}".format(getpass.getuser()))           
```

Comment:
This should be created only once it is really needed. (Otherwise, the tmp dir will be created and not deleted every time thing function runs.)